README.md

```diff
@@ -81,5 +81,5 @@ details see [here](docs/en/contributing/contributing.md)
 
 ## License
 
-[Apache License, Version 2.0](http://www.apache.org/licenses/LICENSE-2.0.html) Copyright 1999-2022 Alibaba Group Holding Ltd.
+[Apache License, Version 2.0](http://www.apache.org/licenses/LICENSE-2.0.html) Copyright (C) Apache Software Foundation.
```

Comment:
@zhongxig can I ask why we reverted this change?

I believe that we need to tell the copyright holder of this project. Refer to the following projects as live examples:

* https://github.com/actions/setup-java/blob/a12e082d834968c1847f782019214fadd20719f6/LICENSE#L4
* https://github.com/google/dagger/tree/b9ed2f0d4d16ceee12fdbf2821e101ddc917ff75#license
* https://github.com/facebook/react/blob/529dc3ce84f0efe99a8be33ff453c09d5801d5ca/LICENSE#L3

The copyright of license is usually described in the license distribution page. In this case at the footer in the [license distribution page](https://www.apache.org/licenses/LICENSE-2.0.html). So we do not need to write its copyright holder here, which may lead readers to misunderstanding.

Reply:
@KengoTODA 
Thanks for the reminder, the description of the readme License has been removed.